src/de/gurkenlabs/litiengine/entities/Creature.java

```diff
@@ -241,21 +237,13 @@ public String toString() {
   }
 
   protected void updateAnimationController() {
-    IEntityAnimationController<?> controller = this.createAnimationController();
+    IEntityAnimationController<?> controller = new CreatureAnimationController<>(this, true);
     this.getControllers().addController(controller);
     if (Game.world().environment() != null && Game.world().environment().isLoaded()) {
       Game.loop().attach(controller);
     }
   }
 
-  protected IEntityAnimationController<?> createAnimationController() {

```

Comment:
Please do not remove this protected method. It is required for overrides in child implementations of `Creature` to create animation controllers at the right time.

Reply:
Hi @nightm4re94 , I added both methods again and restored their usages as they were before.